test/markup/python/decorators.expect.txt

```diff
@@ -1,31 +1,31 @@
 <span class="hljs-meta">@foo</span>
-<span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">bar</span>():</span>
+<span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">bar</span><span class="hljs-params"></span>():</span>
```

Comment:
More so just my curiousity, why is `hljs-params` appearing outside the `()`?

Reply:
Oh I'll take another look, that's not right.

Reply:
Resolved.